encoding/json/outbound.go

```diff
@@ -52,6 +59,9 @@ type Request struct {
 	// Request body. This may be any type that can be serialized by
 	// json.Marshal.
 	Body interface{}
+
+	// Ttlms is the ttl in ms
+	Ttlms int
```

Comment:
Call this `TTL` (all caps for abbreviations) and use the type [`Duration`](https://golang.org/pkg/time/#Duration)